modules/calib3d/src/fundam.cpp

```diff
@@ -909,6 +909,10 @@ void cv::computeCorrespondEpilines( InputArray _points, int whichImage,
     }
 }
 
+inline float scaleFor(float x){
```

Comment:
"static" should be used for local functions.

Reply:
Made the function static